vyper/compiler/__init__.py

```diff
@@ -35,7 +35,11 @@
     # requires assembly
     "abi": output.build_abi_output,
     "asm": output.build_asm_output,
+    "breakpoints": output.build_breakpoints_output,
+    "pc_breakpoints": output.build_pc_breakpoints_output,
     "source_map": output.build_source_map_output,
+    "pc_pos_map": output.build_source_map_output,
```

Comment:
hmm why is this a dup of source_map?

Reply:
I wasn't quite sure how to handle `pc_pos_map` since it is technically already in the `source_map` output. I think what you said below makes sense, so `source_map` output will contain both `pc_pos_map` and `pc_pos_map_compressed`, and there is no need for a separate `pc_pos_map` option. Did I understand it right?

Reply:
right. i'm not sure if we can actually replace the `source_map` option since people might depend on it, but we could add it like `source_map_full` or something

Reply:
Hm, so we would still separate it into `source_map` -> `pc_pos_map_compressed`, and `source_map_full` -> `pc_pos_map`?

Reply:
i think source_map should still return the compressed pc map, but source_map_full should return everything, just like the cli `-f source_map` output.